src/styles/stars.scss

```diff
@@ -0,0 +1,30 @@
+.rate {
+  padding-left: $padding-left;
```

Comment:
The variable `$padding-left` is used here, but it is not defined in this file. Ensure that `$padding-left` is defined somewhere in your SCSS files before it is used.